src/MergeContext.js

```diff
@@ -1126,10 +1126,6 @@ class PullRequest {
     async _doProcess() {
         this._breadcrumbs.push("load");
 
-        /*
-         * Until _loadRawPr(), we must avoid this._rawPr fields except .number.
```

Comment:
I do not think this problem has been addressed in 38574c: It is still possible to access stale `this._rawPr` fields before _loadRawPr() loads fresh PR metadata. I suggest restoring that TODO. I do not recommend implementing that TODO during this project though.

Reply:
> If this TODO's scope is only doProcess() method

No, the problem is not confined to doProcess(). The doProcess() location makes sense IMO because it is there we transition from dangerous inconsistent state to a consistent one, but it is possible that there is a better location for that TODO -- my change request did not consider the location.

> It is still possible, of course, to insert there a dozen of other methods, requiring some other rawPr fields and thus break everything.

Yes, and, somewhat more likely, accidentally access stale rawPr fields when adding code to the _existing_ methods. IIRC, one of your recent commits did exactly that.

> However, following this logic, we would need to add such 'TODOS' in many other places where similar data dependencies take place.

One TODO per object is usually enough. I hope that rawPr is the only stale object that the code may accidentally access, but please correct me if I am wrong. If other such objects exist, then it may be indeed appropriate to flag them with XXXs or TODOs, especially if they stay stale across methods.